docs/lexicon.html

```diff
@@ -186,11 +186,11 @@ <h2 class="title-2" id="python-builtins">
       <span>
         <code class="code"
           ><span class="fn-name">reduce</span><span class="fn-p">(</span
-          ><span class="fn-arg">function</span>, <span class="fn-arg">seq</span
+          ><span class="fn-arg">reducer</span>, <span class="fn-arg">seq</span
```

Comment:
This was copied from here:
https://docs.python.org/3/library/functools.html#functools.reduce

Looks like you've changed the arg name as well which makes it inconsistent with Python if I understand correctly. Should we switch this back?

Reply:
https://github.com/thought-machine/please/blob/8d9ffe8e07e26237f9738c566051d00f953109fd/docs/lexicon.html#L177-L181